lib/winappdriver.js

```diff
@@ -125,14 +126,14 @@ class WinAppDriver {
 
   async start (caps) {
     const executablePath = await getWADExecutablePath();
-    const forcequit =  caps["ms:forcequit"] === true;
+    const isForceQuitEnabled =  caps["ms:forcequit"] === true;
 
     this.process = new WADProcess(this.log, {
       // XXXYD TODO: would be better if WinAppDriver didn't require passing in /wd/hub as a param
       base: DEFAULT_BASE_PATH,
       port: this.proxyPort,
       executablePath: executablePath,
-      forcequit: forcequit
+      isForceQuitEnabled: isForceQuitEnabled
```

Comment:
if key names matches to variable name then it is enough to just do ```
{ 
   isForceQuitEnabled, 
}
```

Reply:
Okay. You mentioned that I should get rid of the intermediate variable, so this won't work. Correct? Should I leave the named argument then?

Reply:
this was just a suggestion. I am happy with the current implementation as it does not have unnecessary duplication